Makefile

```diff
@@ -107,7 +118,7 @@ autoracle-bakerloo: mkdir oracle-server forex-plugins bakerloo-simulator bakerlo
 	@echo "Done building for bakerloo network."
 	@echo "Run \"$(BIN_DIR)/autoracle\" to launch autonity oracle."
 
-autoracle: mkdir oracle-server forex-plugins piccadilly-cax-plugin conf-file e2e-test-stuffs
+autoracle: mkdir oracle-server forex-plugins piccadilly-cax-plugin usdc-plugins conf-file e2e-test-stuffs
```

Comment:
So, building it and making the `piccadilly-cax-plugin` can be removed as legacy.

Reply:
I.e. a `testnet-dax-plugin` dir supercedes this.